app/javascript/src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
 
+import dayjs from "libs/day"; // eslint-disable-line
```

Comment:
I want you to be careful here. I had explicitly pointed out how to handle this import in the Slack channel, and thus I expect you to take care of it. If you are not taking care of it, then there are two possibilities:

- You don't agree with my suggestion or my suggestion doesn't work. Which I am fine with. We can discuss about it and rectify.
- You are still not attending to all Slack notifications. For this we will need to have a more serious talk.

Please do point out why you've written this import statement like this.

Reply:
Yedhin, I did see your message in the slack channel. And I made the corrections accordingly. But since eslint was giving me errors for that line, I added this line: `// eslint-disable-next-line no-unused-vars, unused-imports/no-unused-imports` (vscode suggested to add this line to ignore the eslint for that line). But for some reason that I couldn't understand, the line importing the dayjs was getting removed when I commit the files due to the  `// eslint-disable-next-line no-unused-vars, unused-imports/no-unused-imports` line.
Finally this "removal of line during commit" was fixed by adding `//eslint-disable-line` instead of  `// eslint-disable-next-line no-unused-vars, unused-imports/no-unused-imports`. But when I made this change, I accidentally did auto complete for the dayjs import and it got imported from the lib folder. 

It was my mistake that I didn't catch this during self review. I'll make sure to not make this mistake again.